public/themes/custom/hdbt_subtheme/src/scss/components/content/_unit.scss

```diff
@@ -0,0 +1,94 @@
+$card_spacing: $spacing-and-half;
+
+.unit.unit--high-school-card {
+  margin-top: $spacing-and-half;
+
+  @include breakpoint($breakpoint-s) {
+    padding-left: $card_spacing;
+    width: 50%;
+  }
+
+  @include breakpoint($breakpoint-l) {
+    width: 33.3333%;
+  }
+
+  &:first-child {
+    margin-top: 0;
+  }
+
+  &:nth-child(-n+2) {
+    @include breakpoint($breakpoint-s) {
+      margin-top: 0;
+    }
+  }
+
+  &:nth-child(-n+3) {
+    @include breakpoint($breakpoint-l) {
+      margin-top: 0;
+    }
+  }
+
+  .unit__link {
+    background-color: $color-white;
+    height: 100%;
+    display: block;
+    text-decoration: none;
+
+    &:hover {
+      .unit__title {
+        text-decoration: underline;
+      }
+
+      .unit__arrow {
+        padding-left: $spacing;
+      }
+    }
+  }
+
+  .unit__image {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .unit__text {
+    padding: $spacing-and-half;
+  }
+
+  .unit__title {
+    @include font-size(24px, 28px);
+    font-weight: 500;
+    margin-bottom: 0;
+  }
+
+  .unit__info-row p {
+    margin: 0;
+  }
+
+  .unit__short-desc {
+    @include font-size(18px, 28px);
+    margin-top: $spacing;
+  }
+
+  .unit__arrow {
+    display: block;
+    line-height: 0;
+    // Pull the arrow a bit to the left to align with text.
+    margin: $spacing 0 0 -2px;
```

Comment:
Does this really need resetting the right and bottom margins to work? Where are they set before?

Reply:
copy-paste so no need of the reset - fixing - nice catch!